### PR TITLE
Add emojiWeather

### PR DESCRIPTION
**Pull request by Chandler-Davidson**
Publishing a new plugin for BitBar.

### CODE REVIEW

**Comment by tresni** [comment]
Hi @Chandler-Davidson and thanks for the Pull Request!  We use [jshint](https://www.jshint.com) to lint all Javascript plugins.  It's barfing because you are using some ES6 features (specifically `let`) without saying it should be linted as ES6.  Add this line right under your shebang and it should be good to go:

```js
/* jshint esversion: 6 */
```

File: Weather/emojiWeather.30m.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/usr/local/bin/node
```
Comment (tresni) [inline Weather/emojiWeather.30m.js]: Add jshint configuraiton here to say `esversion: 6`

Comment (Chandler-Davidson) [reply Weather/emojiWeather.30m.js]: Just committed the new line. Do I need to issue a new pull request or is it set? Thanks for your help!

Comment (tresni) [reply Weather/emojiWeather.30m.js]: Nope, should be automagicly picked up here :)

**Comment by tresni** [comment]
Last error is that your script isn't executable (`chmod +x`)  I'll get this one.  Thanks for the Pull Request!

**Pull request merged**